tutorials/shaders/advanced_postprocessing.rst

```diff
@@ -76,19 +76,25 @@ You can also use both options.
 Depth texture
 -------------
 
-To read from the depth texture, perform a texture lookup using ``texture()`` and
-the uniform variable ``DEPTH_TEXTURE``.
+To read from the depth texture, we first need to create a texture uniform set to the depth buffer
+by using ``hint_depth_texture``.
 
 .. code-block:: glsl
 
-  float depth = texture(DEPTH_TEXTURE, SCREEN_UV).x;
+  uniform sampler2D depth_texture : source_color, hint_depth_texture;
+
+Once defined, the depth texture can be read with the ``texture`` function.
```

Comment:
```suggestion
Once defined, the depth texture can be read with the ``texture()`` function.
```
Just a style nitpick. We try to includes parentheses when referencing functions so it is clear at a glance that we are talking about a function and not a variable

Reply:
Oh of course, good catch thank you! :pray: